package.json

```diff
@@ -497,12 +497,32 @@
           "default": "\\"
         },
         "vim.searchHighlightColor": {
-          "type": "string",
-          "description": "Color of the search highlight."
+          "type": [
+            "string",
+            "undefined"
+          ],
+          "description": "Background color of the other search matches. The color must not be opaque so as not to hide underlying decorations. Uses `editor.findMatchHighlightColor` from current theme if undefined."
         },
         "vim.searchHighlightTextColor": {
-          "type": "string",
-          "description": "Color of the search highlight text."
+          "type": [
+            "string",
+            "undefined"
+          ],
+          "description": "Foreground color of the other search matches."
+        },
+        "vim.searchMatchBackgroundColor": {
+          "type": [
+            "string",
+            "undefined"
+          ],
+          "description": "Background color of the current match. Uses `editor.findMatchColor` from current theme if undefined."
```

Comment:
Why does this default to `editor.findMatchColor` rather than `vim.searchHighlightColor`?

Reply:
It would match vscode's own find/replace theming, where the current match uses `editor.findMatchColor`, and other matches use `editor.findMatchHighlightColor` (which `vim.searchHighlightColor` defaults to). There'd be sensible colors for any theme out of the box .